scripts/lib/setup-yum-repo

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+set -x
+set -e
+
+
+yum install -y epel-release
+
+# "Development Tools" is the equivalent of build-essential
+yum groupinstall -y "Development Tools"
+
+yum update
+
+# PostgreSQL 10
+yum localinstall -y "http://yum.postgresql.org/10/redhat/rhel-$(rpm -qf --queryformat="%{VERSION}" /etc/redhat-release)-$(rpm -qf --queryformat="%{ARCH}" /etc/redhat-release)/pgdg-centos10-10-2.noarch.rpm"
```

Comment:
Let's split this command up into a few commands that create variables; this line is way too long :)